<launch>
  <node name="mavlink"
        pkg="rospilot"
        type="mavlink.py"
        respawn="false"
        output="screen"
        args="--baudrate=115200 --device=/dev/ardupilot --udp-export=127.0.0.1:8866" />
  <node name="web_ui"
        pkg="rospilot"
        type="web_ui.py"
        respawn="false"
        output="screen"/>
  <node name="socket_io_server"
        pkg="rospilot"
        type="socket_io_server.py"
        respawn="false"
        output="screen"
        args="--server_port=8080 --policy_port=10843"/>
  <node name="optical_flow"
        pkg="rospilot"
        type="optical_flow.py"
        respawn="false"
        output="screen"
        args="--video_device_num=0"/>
  <node name="position_estimator"
        pkg="rospilot"
        type="position_estimator.py"
        respawn="false"
        output="screen"/>
  <node name="odometry"
        pkg="rospilot"
        type="odometry.py"
        respawn="false"
        output="screen"/>
  <node pkg="tf"
        type="static_transform_publisher"
        name="px4flow_transform"
        args="0 0 0 -1.0471975512 0 3.14159265359 px4flow body 100"/>
  <include file="$(find px4flow)/launch/px4flow.launch"/>
</launch>
